### DISCUSSION TITLE
PFD Dashed route

### DISCUSSION THREAD

**Discussion by franek233233** [flybywiresim/aircraft#6534] [Q&A] [answered]
I warmly welcome

Do you also have such a symptom. 
When you start the A320 FBW which is configured for flight, e.g. EPLL to EGLL through the map configurator (high IFR flight), after the plane loads, the PFD display shows you a dashed line. ?

After starting Auto Pilot, the plane does not fly along this line.
As if the route was not approved in the **FMC**.
Everything is fine in the experimental version, but in the stable version it is.
Am I doing something wrong or is it a BAG.

![image](https://user-images.githubusercontent.com/76412147/149400228-a4d694c5-e4c4-48d7-b45f-4ca5c6260435.png)

**Comment by CptBayer** [comment, marked as answer]
Hi, 

looks like selected HDG mode ... ?! post the FMA the next time, too ... 

greets

**Comment by franek233233** [comment]
I close the topic

I'm a fool :) :)

HDG mode was on
:) :)

Thank you very much.

I would have thought a hundred years and I would not have invented it.

Greetings :)
